### ISSUE TITLE
Discovery: Email notifications from CMS to HelpDesk are not making it through

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#9585]
## Describe the defect
Notifications for upstream data changes to facilities are generating email notifications in the CMS.   However, the email being sent is not being received by VACMSSupport@va.gov

That email address is supposed to forward to support@va-gov.atlassian.net but jira never receives it.

Slack discussion here > https://dsva.slack.com/archives/CT4GZBM8F/p1656018062783369

 The failure is silent.  The CMS has no idea the mail was not received.

## Expected behavior
The email that is sent by the CMS to VACMSSupport@va.gov is received by jira.

## Screenshots
Not received
![image](https://user-images.githubusercontent.com/5752113/175611720-24d6e78d-4e88-4def-9b30-1d3e36e645ad.png)

## Acceptance Criteria

- [x] Determine why emails are failing 
- [x] Create SNOW ticket for support

 ## Additional context

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [x] `⭐️ User support`

**Comment by swirtSJW** [comment]
Carrying this is from the slack thread

If that did not go through to jira, I have a feeling that either :
a)  [VACMSSupport@va.gov](mailto:VACMSSupport@va.gov) is not the right email address
b) the allow list for email that allows your password reset to come through, is not the same allow list that applies to non-mailboxes/forwarders.

**Comment by swirtSJW** [comment]
I think we are going to have to reach out to whoever controls email forwarders for the VA.

**Comment by EWashb** [comment]
@swirtSJW I'm adding this to Sitewide CMS Sprint 64. Since Sitewide CMS introduced this notification we should make sure it is operable for the other teams.

**Comment by EWashb** [comment]
@cmaeng do you have any clues on who to reach out to for email forwards?

**Comment by ndouglas** [comment]
I'm in the middle of writing a SNOW ticket, so if you don't know anyone specific I _might_ have a path forward.

**Comment by cmaeng** [comment]
looking at the email thread from the last time we went through something like this:
- Mike Chellen, PO for Platform CMS team sent the request last time (it can also come from Dave C, just needs to be from a government person)
- Griselda Gallegos, ISSO for va.gov, needs to approve of the change: Griselda.Gallegos@va.gov
- Patrice Milfajt looks to have actually made the change. i think she's a SNOW IT technician so any ESD IT tech can probably do this.

**Mike's request wording:**
"Hi Griselda,

The [VACMSSupport@va.gov](mailto:VACMSSupport@va.gov) email is for users of VA.gov CMS to submit support requests. By policy, all data on CMS is public as it gets published directly to the VA.gov website.

An example of a support email would be:

Hello,

I am working on https://www.va.gov/pittsburgh-health-care/health-services/ and I don’t know if it should match the wording from https://www.saltlakecity.va.gov/services/Emergency_Care.asp.

Can someone help clarify what the wording should be on this page?

Thanks,
Jane Smith
Webmaster, VISN 123

We would like these emails to be forwarded to Jira Service Desk and automatically open tickets, so that CMS staff can track support requests and help the user with their issue.

Please let me know if this helps answer your question, thank you for asking!

-Mike"

**from Griselda's approval email (things she's looking for):**
"Is this an email for notifications ONLY? What kind of data will be included in this notification? Do you have an example of what it would look like?

and after Mike's response:
Thank you for showing me the workflow of how this would be set up. Seeing that there is no PII being included in the notification email and that the purpose would solely be for creating a request in order for the customer to go through the Jira Help Desk versus the VACMSSupport DL, I concur on moving forward."

cc: @EWashb @ndouglas @swirtSJW

**Comment by EWashb** [comment]
Looking at the process that @cmaeng found from the previous iteration of this, the SNOW ticket looks like the first step in the process. @ndouglas I'll wait for your update from SNOW ticket

**Comment by EWashb** [comment]
ACs were not defined. Updated ACs

**Comment by ndouglas** [comment]
I logged into CAG and checked my email, where I found this message:

![Screen Shot 2022-08-04 at 12 03 53 PM](https://user-images.githubusercontent.com/1318579/182894554-73925a2c-734b-4470-b718-f7b6004c241d.png)

So it's been reviewed by a human!  🎉 

![Screen Shot 2022-08-04 at 12 04 37 PM](https://user-images.githubusercontent.com/1318579/182894685-6b85a891-2d04-4f61-9587-bd5f4e6ca274.png)

And I received a response! 🎉 

[Special Alert - VA PIV Legacy CMS Roles Revocation.pdf](https://github.com/department-of-veterans-affairs/va.gov-cms/files/9261275/Special.Alert.-.VA.PIV.Legacy.CMS.Roles.Revocation.pdf)

So, not related to my request.  Maybe I should've been more explicit about what CMS means in this context.

I responded to the ticket trying to clarify who exactly I'm working for.

**Comment by ndouglas** [comment]
No reply to my above ticket; I assume they're just ignored.  I created a new ticket (`REQ7610294`) and explained that I'm working on the Content Management System, not the Card Management System, and that this is related to email forwarding rules (the word "email" occurs ten times in the body of the ticket, as well as the request type itself) and has nothing to do with PIV (which occurs nowhere).

**Comment by cmaeng** [comment]
@ndouglas @EWashb my guess is that the most effective way to get this moving is to contact Griselda directly for approval and then once you have that, let Patrice Milfajt know and she can set up. going through the SNOW ticketing route is going to spin you in circles.

the last time, @ElijahLynn persisted by reaching out through MS Teams and emails until he was able to carve a path forward. my sense is that this is going to need the same type of persistence. give a holler if i can assist.

**Comment by ElijahLynn** [comment]
Discussing in a faster loop here > https://dsva.slack.com/archives/C03LFSPGV16/p1660154940349249

**Comment by ndouglas** [comment]
Should this perhaps have been a Platform CMS issue in the first place?

**Comment by EWashb** [comment]
It looks that way @ndouglas. At first, I was thinking there was just an issue with our setup, but it seems to be bigger than that. I'll discuss with @dawnpruitt

**Comment by EWashb** [comment]
Recapping what we discussed yesterday: it seems like what we need to do is just swap out the emails and it should be fixed, right @ndouglas or @swirtSJW?

**Comment by ndouglas** [comment]
That matches my understanding, but I think @ElijahLynn needs some permissions in Jira to make progress on that.  (I would be in the same boat; I can't even access Jira)

**Comment by EWashb** [comment]
@cmaeng were you the one that needs to swap out that email? I'm trying to remember what we discussed in Scrum to rectify this.

**Comment by cmaeng** [comment]
@EWashb @ndouglas i can do the swap. based on the slack threads i think we're waiting on @ElijahLynn to confirm if he wants to explore other workarounds first. outside of that i'm fairly confident we can test if switching will work.

**Comment by EWashb** [comment]
@cmaeng thanks. My only thing is that this is a sprint goal for us, but the tickets ACs have been met. To make sure we are meeting our obligations I say we close this by EOD in favor for a ticket that explains what the exact approach to the fix is.

**Comment by cmaeng** [comment]
@EWashb a bit of a snag, i'll ping @swirtSJW in slack. VADrupalCMS@va.gov is already designated as the email address for user = api_consumer_chatbot and is being disallowed.

**Comment by swirtSJW** [comment]
We might be able to use the +1 thing like google supports
@cmaeng  I am not sure if outlook supports it, but like
VADrupalCMS@va.gov
VADrupalCMS+1@va.gov
We would need to test sending an email to both and verify they both end up in the same email inbox for VADrupalCMS@va.gov

If that works then we can just set chatbot to VADrupalCMS+chatbot@va.gov  Which would free us up to be able to use VADrupalCMS@va.gov as the master account.

**Comment by cmaeng** [comment]
no bueno: 
![image](https://user-images.githubusercontent.com/60702623/184659961-16d00e54-657a-4c5e-8c71-204fc8675be9.png)

**Comment by EWashb** [comment]
@cmaeng I made a new ticket for that recommendation plus the config changes. I'm going to add your comment above to that ticket and we can work off the new one

**Comment by EWashb** [comment]
Let's work off of #10231

**Comment by EWashb** [comment]
See [slack thread](https://dsva.slack.com/archives/C03LFSPGV16/p1660154940349249) for context
